npe2/manifest/schema.py

```diff
@@ -77,11 +71,6 @@ class PluginManifest(BaseModel):
         regex=r"^[^\W_][\w -~]{1,38}[^\W_]$",
     )
 
-    description: Optional[str] = Field(
-        description="A short description of what your extension is and does."
-        "When unspecified, the description is taken from package metadata."
-    )
```

Comment:
This is the only one that I'm not sure about.  The package description can be a few paragraphs.  Maybe the user wants a shorter/different description for their plugin (e.g. something like napari's builtins). 

We don't actually use the description field anywhere though, so I favor removal.

Reply:
This is replaced by package "summary", which is _not_ the one that is many paragraphs.  See the property below on line 101.

(This was essentially another example of us using a word that didn't match setup cfg)